src/modules/orders/dto/create-order.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+import { IsNotEmpty, IsNumber, IsUUID } from 'class-validator';
+
+export class CreateOrderDto {
+  @ApiProperty({ example: '877f0535-eb91-4159-b51a-7c44ebbc9f37' })
+  @IsUUID()
+  userId: string;
```

Comment:
why you pass it as parameter? its login user, you can take it from token

Reply:
fixed